### DISCUSSION TITLE
about barcode scanner

### DISCUSSION THREAD

**Discussion by dtbj** [saleor/saleor#8901] [Q&A] [answered]
We will have to take orders offline also so is barcode scanner supported by saleor? I mean in our current billing software we can just scan and the product is automatically added to the list.

**Comment by karolkielecki** [comment, marked as answer]
@dtbj Hi. It's not supported. Maybe, in that case, you will be able to extend Saleor to support such functionality by creating an app: https://docs.saleor.io/docs/3.0/developer/extending/apps/key-concepts

**Reply by dtbj** [reply]
Any plans on supporting it?

**Reply by karolkielecki** [reply]
There are no plans to support it.

**Comment by Qodestackr** [comment]
Interesting, @karolkielecki @dtbj what option did you go with?
